liquidcore/site/settings.py

```diff
@@ -90,7 +90,7 @@ def bool_env(value):
 USE_L10N = True
 USE_TZ = True
 STATIC_URL = '/static/'
-STATIC_ROOT = str(base_dir / 'static')
+STATIC_ROOT = str(base_dir / 'liquidcore' / 'home' / 'static')
```

Comment:
I'm not sure this is correct. [`STATIC_ROOT`](https://docs.djangoproject.com/en/2.2/ref/settings/#std:setting-STATIC_ROOT) is meant for deployment, and Django expects to copy the static files over there as part of the deployment steps. So it shouldn't be pointing back to the source code.

Reply:
I've shelled into the container to find out where the static files are actually sitting and it's in that folder. The Dockerfile copies the folder into `./liquidcore`
`ADD liquidcore ./liquidcore`

Reply:
Those are files from one django app. There could, in principle, be more. The `collectstatic` command collects static files from all apps and copies them into a single folder so they can be easily served by e.g. nginx.